erpnext/stock/doctype/item_alternative/item_alternative.json

```diff
@@ -1,12 +1,12 @@
 {
+ "autoname": "field:alternative_item_code",
```

Comment:
Can we not have more than one alternative items linked with an item ?

Reply:
nope